### DISCUSSION TITLE
DSi support?

### DISCUSSION THREAD

**Discussion by MagicalWitchAshley** [profi200/open_agb_firm#148] [General]
I know next to nothing about the work that goes into a project like this, but I've heard that all the DS models have similar hardware that would allow them to support GBA. Is that true, and would that mean something like Open_AGB_Firm would be possible on the DSi?

Regardless, thank you for making a kickass way to revisit the GBA library!

**Comment by profi200** [comment]
It's not possible. They removed all unnecessary GBA hardware on DSi. DS/DS lite can again run them native but only with flashcart. The best you can get on DSi is GBARunner2 at the moment or wait for GBARunner3 (in development).

**Reply by MagicalWitchAshley** [reply]
Aw dang, thanks a ton for the super quick response! 👍

**Reply by RinMaru** [reply]
GBARunner3? why a new one?

**Reply by profi200** [reply]
Because the old one had various issues the new one is solving better. But for details you better ask the author, Gericom.
